docs/backups-storage.md

```diff
@@ -4,7 +4,7 @@ Configure backup storage for your [backup repositories](backups.md#backup-reposi
 
```

Comment:
```suggestion
The required configuration options depend on which cloud storage (or just Persistent Volume) you intend to use for backups:

```

Perhaps mentioning PV one more time will help the reader realize that there is a tab with it hidden at the end.

Reply:
Also, what do you think about adding a subsection in the very beginning (may be on line 3), something like this:

## Set up cloud or local storage for backups

Having a subsection header should improve the local TOC in the right panel

Reply:
But the whole doc is titled Backup storage configuration. It makes little sense to name the subheading the same

Reply:
Any other name? The problem I've found is on the the screenshot 
![image](https://github.com/percona/k8spg-docs/assets/147437/023dfeea-803c-424c-a4df-f27f7d9a0233)